docs/Local-Dispatcher-Windows.md

```diff
@@ -1 +1,126 @@
-# Running a Dispatcher Locally on Windows
+# Running a Dispatcher Locally on Windows IIS
+
+There are a few different versions of Windows and at least 2 ways to set up Dispatcher on them.  The Apache
+Webserver approach is possible, but these instructions are for using the recommended server: _Internet
+Information Services (IIS)_ and leans towards Windows 10.
+
+The instructions largely follow the
+[AEM Documentation](https://experienceleague.adobe.com/docs/experience-manager-dispatcher/using/getting-started/dispatcher-install.html)
+with additional notes, to help avoid problems.
+
+## Host name "aem-publish.local"
+
+Following the AEM instructions will set up your dispatcher on host "localhost" by default.  To
```

Comment:
Should this section come after `Dispatcher Installation Instructions` (below)?

Reply:
No.  If it isn't done before, the reader will do all the changes/configuration to the default site (_localhost:80_ or _127.0.0.1:80_), and then would have to do everything over with the new "_aem-publish.local:8080_" site.